docs/core/docker/intro-net-docker.md

```diff
@@ -0,0 +1,200 @@
+
+* **Low friction install** — You can try out .NET without a local install. Just download a Docker image with .NET in it.
+
+* **Develop in a container** — You can develop in a consistent environment, making development and production environments similar (avoiding issues like global state on developer machines). Visual Studio Tools for Docker even enable you to start a container directly from Visual Studio.
+
+* **Test in a container** — You can test in a container, reducing failures due to incorrectly configured environments or other changes left behind from the last test.
+
+* **Build in a container** — You can build code in a container, avoiding the need to correctly configure shared build machines for multiple environments but instead move to a “BYOC” (bring your own container) approach.
+
+* **Deployment in all environments** — You can deploy an image through all of your environments. This approach reduces failures due to configuration differences, typically changing the image behavior via external configuration (for example, injected environment variables).
+
+[General guidance](https://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/net-core-net-framework-containers/general-guidance) for deciding between .NET Core and .NET Framework for Docker container development.
+
+### Common Docker development scenarios
+
+#### .NET Core
+
+**.NET Core resources**
+
+ Pick the .NET Core samples that fit your scenarios of interest. All sample instructions describe how to target Windows or Linux Docker images from Windows, Linux, or macOS hosts.
+
+The samples use .NET Core 2.0. They use Docker [multi-stage build](https://github.com/dotnet/announcements/issues/18) and [multi-arch tags](https://github.com/dotnet/announcements/issues/14) where appropriate.
+
+* [.NET Core images on DockerHub](https://hub.docker.com/r/microsoft/dotnet/)
+
+* [Dockerize a .NET Core application](https://docs.docker.com/engine/examples/dotnetcore/)
+
+* This .NET Core Docker sample demonstrates how to [use Docker in your .NET Core development process](https://github.com/dotnet/dotnet-docker-samples/tree/master/dotnetapp-dev). The sample works with both Linux and Windows containers.
+
+* This .NET Core Docker sample demonstrates a best practice pattern for [building Docker images for .NET Core apps for production.](https://github.com/dotnet/dotnet-docker-samples/tree/master/dotnetapp-prod) The sample works with both Linux and Windows containers.
+
+* This [.NET Core Docker sample](https://github.com/dotnet/dotnet-docker-samples/tree/master/dotnetapp-selfcontained) demonstrates a best practice pattern for building Docker images for [self-contained .NET Core applications](https://docs.microsoft.com/en-us/dotnet/core/deploying/). Used for the smallest production container without a benefit from [sharing base images between containers](https://docs.docker.com/engine/userguide/storagedriver/imagesandcontainers/). However, lower Docker layers could be shared.
```

Comment:
remove en-us from links.

There are several more below in this same file. All should be removed.

Reply:
@BillWagner - All changes have been addressed and content rebuilt. Thanks for a thorough and thoughtful review!!